models/biocro/R/met2model.BIOCRO.R

```diff
@@ -173,16 +173,16 @@ cf2biocro <- function(met, longitude = NULL, zulu2solarnoon = FALSE) {
   }
   
   ## Convert RH from percent to fraction BioCro functions just to confirm
-  if (met[, max(relative_humidity) > 1]) {
-    met[, `:=`(relative_humidity = relative_humidity/100)]
+  if (met[, max(met$relative_humidity) > 1]) {
+    met$relative_humidity = met$relative_humidity/100
   }
   newmet <- met[, list(year = lubridate::year(date),
                        doy = lubridate::yday(date),
                        hour = round(lubridate::hour(date) + lubridate::minute(date) / 60, 0),
                        SolarR = ppfd, 
-                       Temp = udunits2::ud.convert(air_temperature, "Kelvin", "Celsius"), 
-                       RH = relative_humidity, 
+                       Temp = udunits2::ud.convert(met$air_temperature, "Kelvin", "Celsius"),
```

Comment:
Is the `met$` necessary? Why is it used for some variables but not others (like wind_speed and ppfd)?